mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -1098,3 +1098,6 @@ def get_model_version_by_alias(self, name, alias):
                 raise MlflowException(
                     f"Registered model alias {alias} not found.", INVALID_PARAMETER_VALUE
                 )
+
+    def _await_model_version_creation(self, mv, await_creation_for):
+        pass
```

Comment:
Is this change really correct?

Reply:
ah I see

Reply:
can we insert a comment on why we don't wait?

Reply:
Will do